transports/rabbitmq/index.md

```diff
@@ -26,6 +26,15 @@ snippet: rabbitmq-config-basic
 The RabbitMQ transport requires a connection string to connect to the broker. See [Connection settings](/transports/rabbitmq/connection-settings.md) for options on how to provide the connection string.
 
 
+### Topology selection
+
+Topologies are used to control how native Rabbit constructs like queues and exchanges are created. In Versions 5 and above, selecting a topology is mandatory. For backwards compatiblility use the `ConventionalRoutingTopology`, which was the previous default:
+
+snippet: rabbitmq-config-topology
```

Comment:
It looks like the build is failing because it can't find this snippet. Does it have to be in the `Rabbit_All` project instead of the `Rabbit_5` one?

Reply:
Think we need to pull it out to a separate partial since it only applies to v5?

Reply:
I've pushed a fix